pyproject.toml

```diff
@@ -61,25 +63,25 @@ gdspy = {version="*", optional = true}
 gdstk = [
     {version="*", optional = true, platform = "linux"},
     {version="*", optional = true, platform = "win32"},
-    {version="<0.9.38", optional = true, platform = "darwin"},
+    {version=">=0.9.49", optional = true, platform = "darwin"},
```

Comment:
I think we can just make this `>=0.9.49` across the board maybe? unless it's useful to support all versions for windows / linux

Reply:
Done